### DISCUSSION TITLE
Is there a way to partially use a store injected by a Provider?

### DISCUSSION THREAD

**Discussion by livemehere** [pmndrs/jotai#2676] [Q&A] [answered]
As the title suggests.

For example, consider the following situation:
There is an atom that manages the global state, and there is an atom that I want to use only within a specific page (let’s call this atom A, which is similar to the Context API).

Atom A is used within a Provider and store to ensure that it is automatically initialized whenever entering the specific page and is only accessible within that page.

The issue arises because the store injected by the Provider does not include the global atom, making the global state inaccessible within this Provider.

Is there a usage method I am unaware of? If not, I am considering making a PR.

Let me know if you need any more assistance!

**Comment by rothsandro** [comment, marked as answer]
Navigating away from the page should clean up the state of atom A, right? If yes, you can either

- create the atom inside the page component and share the atom via React Context
- use Jotai Scope which lets you scope atoms https://jotai.org/docs/extensions/scope#jotai-scope

**Reply by livemehere** [reply]
Thank you so mush.
After reading this, I think using 'scope' is a mush more appropriate way to do it than using 'store'.
The purpose that store aims to provide is somewhat different that of scope.
